### DISCUSSION TITLE
unsupervised multivariate TS anomaly detection

### DISCUSSION THREAD

**Discussion by andreaceruti** [TDAmeritrade/stumpy#918] [Q&A] [answered]
Hey, sorry if this is a stupid question but I am new to matrix profile algorithms.

My task is to find anomalies from unsupervised multivariate (10 dimensions) time series data coming from industrial sensors. I have read your documentation but I did not find any tutorial about this and so I am having some difficulties.

Is it possible to have some references about anomaly detection on multivariate and unsupervised time series data, so that I can try your algorithms? I am currently using autoencoders and they works fine, but I was wondering if matrix profile could overcome the results I have

**Comment by seanlaw** [comment, marked as answer]
@andreaceruti Thank you for your question and welcome to the STUMPY community. So, in the matrix profile world, "potential anomalies" are often referred to as a "discord". If you truly have multi-dimensional data (i.e., multiple time series that have data points captured at the same time), then I recommend going over the [multi-dimensional motif discovery tutorial](https://stumpy.readthedocs.io/en/latest/Tutorial_Multidimensional_Motif_Discovery.html) but, more specifically, take a look at the `discords=True` parameter as mentioned [here in the tutorial](https://stumpy.readthedocs.io/en/latest/Tutorial_Multidimensional_Motif_Discovery.html#Constrained-Search). Please be warned that multi-dimensional matrix profiles are super tricky and can be hard to interpret so you may be on your own. Without knowing your problem and its associated context, it is very hard for us to know what may/may not work but I hope this helps.
